components/Page/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Head from 'next/head';
+import { Layout } from 'antd';
+import Header from '../Header';
+const { Content, Footer } = Layout;
+
+type LayoutProps = {
+	title?: string
+	children?: any
+}
+
+
+const Page = (props: LayoutProps): JSX.Element => (
+	<>
+		<Head>
```

Comment:
Instead of importing the Page component in every page, create a Layout file and import Header in that. Then wrap the component of _app.js with this Layout component. This will be needed for setting up redux too so it's better to do it this way.

Reply:
Makes sense, I'll change it.

Reply:
Also, to set the title for every page, you can ```import Head from 'next/head;``` and use it. Checkout this [link](https://nextjs.org/docs/api-reference/next/head) for more info.

Reply:
How do I set a custom title for every page when the head is in the _app.tsx file?

Reply:
No no, you can import the Head component in whichever page you need to set the title and set it as needed. You don't need to pass the title as an argument it is being done with page Component in the current version is what I meant.

Reply:
I'd recommend creating a seperate Layout component, import ANTD's Layout in there, set up `next/head` in there, and set a prop in the custom Layour component to take in the title for the page.

Reply:
But this Layout component is serving as a wrapper for the Main Component in _app.js.
So how would every page be able to pass the title as an argument ? 

Or do you mean 2 layout components, one as the wrapper for the entire app and another component which is used as a wrapper for every page to set the title ?